### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by Lilmatt6969** [community/community#79304] [Discover: GitHub Best Practices]
Me

**Comment by sldimitrov** [comment]
Hello there! I suggest that you're new here but if you pin my comment as an answer to your 'question' you could help me much!

**Reply by queenofcorgis** [reply]
@sldimitrov GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers.  Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by ghost** [comment]
Hi @Lilmatt6969 looks like you are new to github. Let me just list out the things you should do and look out for.

Contribute or Create Projects - You can either start contributing to existing projects by forking repositories and making pull requests or start your own projects by adding files, writing code, and collaborating with others(Looks like you already created a repo).

Explore GitHub Features - GitHub offers a lot of features like issues, pull requests, project boards, actions, etc. Spend some time exploring these functionalities to understand how they can help in project management and collaboration.

Join Communities - GitHub has a vast community. You can find projects to contribute to or people to collaborate with based on your interests. Explore different repositories and engage in discussions(And make sure you follow its guidelines ).

Documentation and Help - If you're ever stuck or need guidance, refer to their guides or even search online for tutorials or forums where people discuss GitHub-related topics.

Have Fun and Learn - GitHub is a fantastic platform not just for version control but for learning, collaboration, and showcasing your work. Enjoy the experience and don't hesitate to ask questions or seek guidance as you explore.

Things you should learn:
- Commits
- Repositories
- Branches
- Pull Requests
- Issues
- Contributions/Forking
- Documentation on README.md
- Git (If you want to be efficient)

Make sure you contibute and commit often to reach the heights in your journey. If This helped you, mark mine as answer

All The Best
